app/views/IfYouvePaidYourVATPage.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        layout: Layout,
+        link: components.link,
+        pageHeader: components.pageHeader
+)
+
+@()(implicit messages: Messages, appConfig: AppConfig, user: UserRequest[_])
+
+@layout(
+    pageTitle = messages("paidYourVat.headingAndTitle"),
+    isPageFullWidth = false,
+    backLinkEnabled = false
```

Comment:
I've set this to False as the page doesn't currently have a page to go back to and so throws up an error. This can be set to true when everything is linked up.